constants/resources.py

```diff
@@ -1,2 +1,7 @@
 OADP_API_GROUP = "oadp.openshift.io"
-VELERO_API_GROUP = "velero.io"
\ No newline at end of file
+VELERO_API_GROUP = "velero.io"
+STATUS_UNAVAILABLE = "Unavailable"
```

Comment:
maybe add a file status.py here under constants and put these statuses under a Enum class?
example:
https://gitlab.cee.redhat.com/app-mig/mtc-python-client/-/blob/master/src/mtc/enums/migmigration_enums.py#L6
We will probably have multiple constants in the future, also different statuses per resource, so it better to group it under a Enum class IMO